pyrgg/pyrgg.py

```diff
@@ -208,7 +208,7 @@ def input_filter(input_dict):
         filtered_dict["self_loop"] = 1
     if filtered_dict["multigraph"] not in [1, 2]:
         filtered_dict["multigraph"] = 1
-    if filtered_dict["output_format"] not in list(range(1, 12)):
+    if filtered_dict["output_format"] not in list(range(1, 13)):
```

Comment:
Can't we use `len(SUFFIX_MENU)` here instead of 13?

Reply:
You are right ;-)
Solved in https://github.com/sepandhaghighi/pyrgg/pull/38/commits/dee980ba42c8ce714b9864e740f0f56a1447cb0e

Reply:
Ok, I'll handle it in #39 .